uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorDataFormatTest.java

```diff
@@ -306,14 +308,23 @@ private void assertDataFormatPath(boolean expected) {
 			assertEquals("DataFormat - " + dataFormatName, expected, wait.getResult());
 		}
 	}
-
-	/**
-	 * https://issues.jboss.org/browse/FUSETOOLS-1930
-	 */
-	private boolean testIssue_1930() {
-		if (dataFormat.contains("zipfile")) {
-			return true;
+	
+	/*
+	 * Checks if JIRA issue is opened. If yes, checks if actual parameter is reported one.
+	 * https://issues.redhat.com/browse/FUSETOOLS-1930
+	 * @return true if parameter is still reported with issue
+	 * @return false else
+	 * */
+	public boolean checkForReportedParameters() {
+		if (new JiraClient().isIssueClosed("FUSETOOLS-1930")) {
+			return false;
+		} else {
+			if (dataFormat.equals(
+					"zipfile - The Zip File data format is a message compression and de-compression format of zip files.")) {
+				return true;
+			} else {
+				return false;
+			}
```

Comment:
```suggestion
			return "zipfile - The Zip File data format is a message compression and de-compression format of zip files.".equals(dataformat);
```

it avoids potential NPE if dataformat is null.
less line of code.

Reply:
Good point, changed. Thanks!